packit/distgit.py

```diff
@@ -406,6 +408,48 @@ def build(
         )
         pkg_tool.build(scratch=scratch, nowait=nowait, koji_target=koji_target)
 
+    def get_tag_latest_build(self, downstream_package_name, dist_git_branch):
```

Comment:
This name implies to me getting a tag, can we rename it to something like `get_latest_build_in_tag`?